routes/payment_intent/index.mjs

```diff
@@ -0,0 +1,297 @@
+//@ts-check
+import { Hono } from "hono";
+import { createSupabaseClient } from "../../utils/supabase.mjs";
+const app = new Hono();
+
+// Initialize Supabase client
+const supabase = createSupabaseClient();
+
+//create payment intent
+app.post("/", async (c) => {
+  const { from_number } = await c.req.json().catch(() => ({}));
+
+  if (!from_number) {
+    return c.json(
+      {
+        message: "Invalid request; missing `from_number`",
+      },
+      400
+    );
+  }
+
+  try {
+    // @ts-ignore
+    const { data, error } = await supabase
+      .from("intents")
+      .insert({
+        from_number: from_number,
+        client_secret: `client_secret_${from_number}`,
+      })
```

Comment:
would be nice to have a created time. I wonder if this is something we can add automatically on the supabase end

Reply:
the created time is done automatically on supabase